src/pages/_app.tsx

```diff
@@ -1,5 +1,7 @@
 import React from "react";
-import "bootstrap/scss/bootstrap.scss"; // we should replace this with our own SASS file
+
+// custom SASS file
+import "../styles/index.scss";
 import { AppProps } from "next/app";
 import { ThemeProvider } from "styled-components";
 
```

Comment:
In this file, there should be a theme object. It would be nice if the colors from the sass file are duplicated in this object so that it can be referenced in styled-components that we use